cmd/ako-main/main.go

```diff
@@ -166,7 +166,11 @@ func InitializeAKC() {
 	if err != nil {
 		utils.AviLog.Warnf("Error in creating openshift clientset")
 	}
-
+	if lib.GetCNIPlugin() == lib.OPENSHIFT_CNI || lib.GetCNIPlugin() == lib.OVN_KUBERNETES_CNI {
```

Comment:
Shall we club these two statements since both are related?

Reply:
addressed